doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc

```diff
@@ -0,0 +1,361 @@
+|Values
+|Default
+|Description
+
+|`kie.maven.settings.custom`
+|Path
+|N/A
+|The location of a custom `settings.xml` file for Maven configuration.
+
+|`kie.server.jms.queues.response`
+|String
+|`queue/KIE.SERVER.RESPONSE`
+|The response queue JNDI name for JMS.
+
+|`org.drools.server.filter.classes`
+|`true`, `false`
+|`false`
+|When set to `true`, the Drools {KIE_SERVER} extension accepts custom classes annotated by the `XmlRootElement` or `Remotable` annotations only.
+
+ifdef::PAM,JBPM[]
+|`org.kie.server.bypass.auth.user`
+|`true`, `false`
+|`false`
+|A property that enables you to bypass the authenticated user for task-related operations, for example queries.
+
+|`org.jbpm.rule.task.firelimit`
+|Integer
+|`10000`
+|A property that specifies maximum number of fired rules to avoid situation where rules run into infinite loop and make server completely unresponsive.
+
+|`org.jbpm.ejb.timer.local.cache`
+|`true`, `false`
+|`true`
+|This property turns off the EJB Timers local cache.
+endif::[]
+
+|`org.kie.server.domain`
+|String
+|N/A
+|The JAAS `LoginContext` domain used to authenticate users when using JMS.
+
+|`org.kie.server.repo`
+|Path
+|`.`
+|The location where {KIE_SERVER} state files will be stored.
+
+|`org.kie.server.sync.deploy`
+|`true`, `false`
+|`false`
+|A property that instructs the {KIE_SERVER} to hold the deployment until the {CONTROLLER} provides the containers deployment configuration. This property only affects servers running in managed mode. The options are as follows:
+
+* `false`; the connection to the {CONTROLLER} is asynchronous. The application starts, connects to the {CONTROLLER}, and once successful, deploys the containers. The application accepts requests even before the containers are available.
+* `true`; the deployment of the server application joins the {CONTROLLER} connection thread with the main deployment and awaits its completion. This option can lead to a potential deadlock in case more applications are on the same server instance. It is recommended that you use only one application (the server) on one server instance.
+
+|`org.kie.server.startup.strategy`
+|`ControllerBasedStartupStrategy`, `LocalContainersStartupStrategy`
+|`ControllerBasedStartupStrategy`
+|The Startup strategy of {KIE_SERVER} is used to control the KIE containers that are deployed and the order in which they are deployed.
+
+|`org.kie.server.mgmt.api.disabled`
+|`true`, `false`
+|`false`
+|When set to `true`, disables {KIE_SERVER} management API.
+
+|`org.kie.server.xstream.enabled.packages`
+|Java packages like `org.kie.example`. You can also specify wildcard expressions like `org.kie.example.*`.
+|N/A
+|A property that specifies additional packages to whitelist for marshalling using XStream.
+
+|`org.kie.store.services.class`
+|String
+|`org.drools.persistence.jpa.KnowledgeStoreServiceImpl`
+|Fully qualified name of the class that implements `KieStoreServices` which are responsible for bootstraping KieSession instances.
```

Comment:
"that are responsible for bootstrapping `KieSession` instances."  Correcting some typos and formatting.